HIP/hip-583.md

```diff
@@ -10,220 +10,452 @@ status: Accepted
+| ANY              | None                        | `account-num alias`         |
+| ANY              | ECDSA secp256k1 `key alias` | `ECDSA` derived evm address |
+| ANY              | `evm address alias`         | `evm address alias`         |
+
+### Update Account with Alias
+
+[HIP 32](https://hips.hedera.com/hip/hip-32) specifies:
+> An account created normally has no alias. It can be given an alias with an account update, but only if that alias is
+> not currently used by any other account, and only if the update transaction is signed by the private key corresponding
+> to the public key in the alias.
+
+This HIP **SUPERSEDES** `HIP-32` by removing this capability. Support for updating an account with an `alias` introduces
+significant complication. If an account without an alias has already interacted with a smart contract, then it is known
+to the smart contract with the "long-zero" `account-num alias` that all accounts have. If that account were to be
+updated with an alias, then which evm address should be used in the EVM? For compatibility, it should use the
+`account-num alias`, at least with contracts it had already interacted with. However, this may run contrary to some
+expectations. While this is an important user flow we want to support, we defer this support to a future HIP. A full
+specification for this capability will likely require the ability for the user to select which alias is their default,
+and to specify on a per-call basis the alias they want to use when interacting with a smart contract.
+
+User stories
+------------------------------------------------------------------------------------------------------------------------
+
+1. As a user with an `ECDSA` derived account from another chain who uses an Ethereum-compatible wallet, I would like to
+   create my Hedera account using that same ECDSA public key and associated Ethereum compatible `evm address alias`.
+2. As a developer, I would like to create a new account with a `key alias` via the `CryptoCreate` transaction.
+3. As a developer, I would like to create a new account with an `evm address alias` via the `CryptoCreate` transaction.
+4. As a developer, I would like to transfer HBAR or tokens to a Hedera account using their `evm address alias`.
+5. As a Hedera user with an `Ethereum-native wallet`, I would like to receive HBAR or tokens in my account by sharing
+   only my `evm address alias`.
+6. As a Hedera user with an `Ethereum-native wallet`, I would like to send HBAR or tokens from my account to another
+   account on Hedera using a valid 20-byte address.
+7. As a Hedera user with a `Hedera-native wallet`,  I would like to transfer HBAR or tokens to another account using
+   only the recipient's `evm address alias`.
+8. As a user with an Ethereum-native wallet, I want to be able to only share my EVM public address to an exchange, and
+   that exchange to create a hollow account for me by transferring funds to the address.
+
+Specification
```

Comment:
We need a section that discusses throttles. How many hollow account creations can we do per second? Do hollow account creations count against account creation throttle bucket? When we upgrade a hollow account to a full account, does it count against the account update throttle? Or account creation throttle? This needs to be specified.